orange/src/main/java/com/hartwig/hmftools/orange/algo/purple/GermlineHeterozygousDeletionFactory.java

```diff
@@ -0,0 +1,69 @@
+package com.hartwig.hmftools.orange.algo.purple;
+
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.collect.Maps;
+import com.hartwig.hmftools.common.ensemblcache.EnsemblDataCache;
+import com.hartwig.hmftools.common.gene.TranscriptData;
+import com.hartwig.hmftools.common.purple.GeneCopyNumber;
+import com.hartwig.hmftools.common.purple.GermlineDeletion;
+import com.hartwig.hmftools.common.purple.GermlineStatus;
+import com.hartwig.hmftools.datamodel.purple.GeneProportion;
+import com.hartwig.hmftools.datamodel.purple.ImmutablePurpleHeterozygousDeletion;
+import com.hartwig.hmftools.datamodel.purple.PurpleHeterozygousDeletion;
+
+import org.jetbrains.annotations.NotNull;
+
+public class GermlineHeterozygousDeletionFactory
+{
+    @NotNull
+    private final EnsemblDataCache ensemblDataCache;
+
+    public GermlineHeterozygousDeletionFactory(@NotNull final EnsemblDataCache ensemblDataCache)
+    {
+        this.ensemblDataCache = ensemblDataCache;
+    }
+
+    @NotNull
+    public Map<PurpleHeterozygousDeletion, GermlineDeletion> mapDeletions(@NotNull List<GermlineDeletion> germlineDeletions,
+            @NotNull List<GeneCopyNumber> allSomaticGeneCopyNumbers)
+    {
+        Map<PurpleHeterozygousDeletion, GermlineDeletion> deletionMap = Maps.newHashMap();
+        for(GermlineDeletion germlineDeletion : germlineDeletions)
+        {
+            if(germlineDeletion.TumorStatus == GermlineStatus.HET_DELETION)
```

Comment:
GermlineDeletion has both a NormalStatus and a TumorStatus. As far as I know the names of these fields accurately describe their meaning. Perhaps we shouldn't use the GermlineStatus enum for this, since that obfuscates the fact that we are looking at the status in the tumor. But that would require changes outside of Orange or the datamodel. 
The name GermlineHeterozygousDeletion maybe also doesn't help, since that makes it look like the heterozygosity is about the germline, not the tumor. Perhaps GermlineLossOfHeterozygosity, or something?

Reply:
You are right, the name of the enum is the cause for confusion. Don't think important enough to change though (the logic is correct).

I like the suggestion to rename to `GermlineLossOfHeterozygosity`, feels more consistent with other names in ORANGE and more descriptive indeed.

Related - shouldn't your selection be different? Shouldn't germline heterozygous deletions be "normal status = HET_DELETION and tumor status != HOM_DELETION"? Or would that practically be the same as tumor status = HET_DELETION anyways? I guess a germline deletion implies at least "normal status = HET_DELETION" anyways?